src/findMailboxId.ts

```diff
@@ -1,7 +1,11 @@
 import { Db } from './database';
 
-export const findMailboxId = (database: Db, repoId: number): number | null => {
-  const [mailboxId = null] = database.data.find(([_, ids]) => ids.some(id => id === repoId)) || [];
+export const findMailbox = (
+  database: Db,
+  repoId: number
+): { mailboxId: number | null; assignTo?: number } => {
```

Comment:
We can return `null` or an `object` rather than always an object. The condition will be simpler for the caller.